CONTRIBUTING.md

```diff
@@ -57,6 +57,13 @@ cd fiddle
 yarn
 ```
 
+Optional: Install [`husky`](https://typicode.github.io/husky) and enable
```

Comment:
In the [`husky` migration guide](https://dev.to/typicode/what-s-new-in-husky-5-32g5#article-body:~:text=You%20can%20keep%20the%20previous%20behaviour%20(recommended)%20by%20adding%20a%20postinstall%20script%3A), they recommend keeping the old behaviour with a `postinstall` script.

Any reason we switched to the optional install?

Reply:
I don't have strong feelings either way, let me add it back in.